### ISSUE TITLE
slide crashed while trying to save an image

### ISSUE THREAD

**Issue by Fernando-UofT** [ccrama/Slide#230]
## Info
try to save  imagen in  https://www.reddit.com/r/ImaginaryLandscapes/comments/3pqsg8/from_arrietty/
## Stacktrace
```
java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String java.io.File.getAbsolutePath()' on a null object reference
at me.ccrama.redditslide.Activities.FullscreenImage.saveImageGallery(FullscreenImage.java:206)
at me.ccrama.redditslide.Activities.FullscreenImage.access$100(FullscreenImage.java:36)
at me.ccrama.redditslide.Activities.FullscreenImage$5$1.onLoadingComplete(FullscreenImage.java:150)
at com.nostra13.universalimageloader.core.DisplayBitmapTask.run(DisplayBitmapTask.java:72)
at android.os.Handler.handleCallback(Handler.java:739)
at android.os.Handler.dispatchMessage(Handler.java:95)
at android.os.Looper.loop(Looper.java:135)
at android.app.ActivityThread.main(ActivityThread.java:5254)
at java.lang.reflect.Method.invoke(Native Method)
at java.lang.reflect.Method.invoke(Method.java:372)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:903)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:698)
```
Slide v4.2.1, API v22
Reported via Slide Crash Capture

**Comment by Alexendoo** [comment]
I didn't experience a crash when trying it however slide did take a good while to process it, and android gave me the option to kill/wait Slide. Something to look into

**Comment by ccrama** [comment]
Fixed eefdfd5ecef9573fc098c79af87c7bd61272565d

**Comment by Fernando-UofT** [comment]
Thanks for all the effort you put on this

On 23 October 2015 10:11:01 GMT-04:00, ccrama <notifications@github.com> wrote:
>Fixed eefdfd5ecef9573fc098c79af87c7bd61272565d
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/ccrama/Slide/issues/230#issuecomment-150583660

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by ccrama** [comment]
No problem! Thank you guys for the bug reports :)
